src/interactions/interaction.ts

```diff
@@ -24,17 +25,22 @@ export class Interaction {
    * @returns {Interaction} The calling Interaction.
    */
   public attachTo(component: Component) {
-    if (this._componentAttachedTo) {
-      this.detachFrom(this._componentAttachedTo);
-    }
-
+    this._detach();
     this._componentAttachedTo = component;
-    component.onAnchor(this._anchorCallback);
-
+    this._attach();
     return this;
   }
 
   /**
+   * Attaches to existing _componentAttachedTo if enabled.
```

Comment:
The variable is `protected` so we can't change the name without causing a break.

Technically it's attached to the Component, just not listening...